src/main/groovy/se/diabol/scrolls/Scrolls.groovy

```diff
@@ -0,0 +1,94 @@
+package se.diabol.scrolls
+
+class Scrolls {
+    static void main(String[] args) {
+        def options = parseOptions(args)
+        if (!options) {
+            return
+        }
+
+        def config
+        def plugins
+        try {
+            config = readConfig(options.config)
+            plugins = initializePlugins(config)
+        } catch (RuntimeException e) {
+            println "ERROR: ${e.message}, aborting..."
+            return
```

Comment:
SCL-2: Scrolls should exit with non-zero exit code when it can not continue
created for this